packages/nextclade/src/qc/qc_config.rs

```diff
@@ -233,6 +233,27 @@ impl QcRulesConfigStopCodons {
   }
 }
 
+/// Configuration for QC rule "recombinants"
+#[derive(Clone, Debug, Default, Eq, PartialEq, Serialize, Deserialize, schemars::JsonSchema, Validate)]
+#[serde(rename_all = "camelCase")]
+#[serde(default)]
+#[schemars(example = "QcRulesConfigRecombinants::example")]
+pub struct QcRulesConfigRecombinants {
+  pub enabled: bool,
+  pub mutations_threshold: usize,
+  pub score_weight: OrderedFloat<f64>,
+}
+
+impl QcRulesConfigRecombinants {
+  pub const fn example() -> Self {
+    Self {
+      enabled: true,
+      mutations_threshold: 20,
+      score_weight: OrderedFloat(100.0),
+    }
+  }
+}
```

Comment:
Configurable variables can be adjusted here (maps to fields in `qc.recombinants` of `pathogen.json`). They are passed to the function which calculates the metric.